api/v1alpha1/edgedevice_types.go

```diff
@@ -34,10 +34,23 @@ type EdgeDeviceSpec struct {
 	// RequestTime is the time of device registration request
 	RequestTime *metav1.Time `json:"requestTime,omitempty"`
 
-	Heartbeat     *HeartbeatConfiguration         `json:"heartbeat,omitempty"`
-	Storage       *Storage                        `json:"storage,omitempty"`
-	Metrics       *MetricsConfiguration           `json:"metrics,omitempty"`
-	LogCollection map[string]*LogCollectionConfig `json:"logCollection,omitempty"`
+	Heartbeat       *HeartbeatConfiguration         `json:"heartbeat,omitempty"`
+	Storage         *Storage                        `json:"storage,omitempty"`
+	Metrics         *MetricsConfiguration           `json:"metrics,omitempty"`
+	LogCollection   map[string]*LogCollectionConfig `json:"logCollection,omitempty"`
+	MetricsReceiver *MetricsReceiverConfiguration   `json:"metricsReceiver,omitempty"`
```

Comment:
Please move it under `Metrics` to keep metrics configuration in one place.

Reply:
done